browser/result_set.go

```diff
@@ -0,0 +1,27 @@
+package browser
+
+type FileData struct {
```

Comment:
I do suggest exporting the structures... but please give them a comment. If you run the metalinter it'll tell you to do that too :)